.eslintrc.json

```diff
@@ -15,7 +15,6 @@
           "error",
           {
             "type": "attribute",
-            "prefix": "app",
```

Comment:
Czemu?

Reply:
to jest guard na dyrektywy, żeby były napisane sensownie - tylko pisanie dyrektyw z prefixem app nic nie wnosi. Uważam, że to mało mądre, żeby tak było.